src/hooks/useInventoryBrowse/constants.js

```diff
@@ -22,6 +23,17 @@ export const PATH_MAP = {
   [browseModeOptions.CONTRIBUTORS]: 'browse/contributors/instances',
 };
 
+export const _PATH_MAP = {
```

Comment:
Same about indexes here - we should add some explanation why there are two almost identical constants